VOKMockUrlProtocol.m

```diff
@@ -271,42 +296,49 @@ - (VOKMockUrlProtocolResponseAndDataContainer *)responseAndDataFromHTTPResponseS
-        DLOG(@"Serving mock data from complete HTTP response file (%@)", filePath);
-        return [self responseAndDataFromHTTPResponseString:fileContents];
+    for (NSString *resourceName in resourceNames) {
+        filePath = [[NSBundle bundleForClass:[self class]] pathForResource:resourceName
+                                                                    ofType:@"http"
+                                                               inDirectory:MockDataDirectory];
+        NSString *fileContents = [NSString stringWithContentsOfFile:filePath
+                                                           encoding:NSUTF8StringEncoding
+                                                              error:NULL];
+        if (fileContents) {
+            // We've got a complete-HTTP-response file, so parse it.
+            DLOG(@"Serving mock data from complete HTTP response file (%@)", filePath);
+            return [self responseAndDataFromHTTPResponseString:fileContents];
+        }
     }
     
     // Otherwise, look for a JSON data file.
-    filePath = [[NSBundle bundleForClass:[self class]] pathForResource:resourceName
-                                                                ofType:@"json"
-                                                           inDirectory:MockDataDirectory];
-    NSData *data = [NSData dataWithContentsOfFile:filePath];
-    if (data) {
-        // We've got a JSON data file, so send it.
-        DLOG(@"Serving mock data from JSON response body file (%@)", filePath);
-        NSHTTPURLResponse *response = [[NSHTTPURLResponse alloc] initWithURL:self.request.URL
-                                                                  statusCode:kHTTPStatusCodeOK
-                                                                 HTTPVersion:@"HTTP/1.1"
-                                                                headerFields:@{
-                                                                               @"Content-type": @"text/json",
-                                                                               }];
-        return [VOKMockUrlProtocolResponseAndDataContainer containerWithResponse:response
-                                                                            data:data];
+    for (NSString *resourceName in resourceNames) {
+        filePath = [[NSBundle bundleForClass:[self class]] pathForResource:resourceName
+                                                                    ofType:@"json"
+                                                               inDirectory:MockDataDirectory];
+        NSData *data = [NSData dataWithContentsOfFile:filePath];
+        if (data) {
+            // We've got a JSON data file, so send it.
+            DLOG(@"Serving mock data from JSON response body file (%@)", filePath);
+            NSHTTPURLResponse *response = [[NSHTTPURLResponse alloc]
+                                           initWithURL:self.request.URL
```

Comment:
That's the correct alignment (left-aligned rather than `:`-aligned) for when there's a line break between the target of the message and the first part of the message-selector.